### PR TITLE
Apply formatting rules.

### PR DESCRIPTION
**Pull request by ItzLevvie**

### CODE REVIEW

File: src/GameServer.js [hunk 1]
```diff
@@ -1215,11 +1215,7 @@ GameServer.prototype.splitCells = function(client) {
         // Create cell
         var split = new Entity.PlayerCell(this.getNextNodeId(), client, startPos, newMass, this);
         split.setAngle(angle);
-        // Polyfill for log10
-        Math.log10 = Math.log10 || function(x) {
-            return Math.log(x) / Math.LN10;
-        };
-        var splitSpeed = this.config.splitSpeed * Math.max(Math.log10(newMass) - 2.2, 1); //for smaller cells use splitspeed 150, for bigger cells add some speed
+        var splitSpeed = this.config.splitSpeed * Math.max((Math.log(newMass) / 2.3) - 2.2, 1); //for smaller cells use splitspeed 150, for bigger cells add some speed
```
Comment (ItzLevvie) [inline src/GameServer.js]: This is a shortened version of the Math.log10

**Comment by Andrews54757** [comment]
Im not sure what that portion of code does on log10.

**Comment by ItzLevvie** [comment]
@Andrews54757 That "Polyfill for log10" replaces Math.log10 to Math.log.

**Comment by Andrews54757** [comment]
I know but the problem is I didn't add that code, someone else did because
they were having errors on their computer because of the older one but he
created a polyfill for it. (where it works on all)

**Comment by Andrews54757** [comment]
Later please, i am doing serious updates so any formatting done might be undone

**Pull request closed without merging**